databricks/koalas/tests/test_groupby.py

```diff
@@ -80,6 +80,10 @@ def test_groupby(self):
 
         self.assertRaises(TypeError, lambda: kdf.a.groupby(kdf.b, as_index=False))
 
+        self.assertRaises(ValueError, lambda: kdf.groupby('a', axis=1))
```

Comment:
so should be fixed here also

Reply:
Could you add tests to specify `axis=0` and `'index'`?

Reply:
@itholic @ueshin okay let me fix it. thank you!